openstack/common/tags/requests.go

```diff
@@ -12,8 +12,8 @@ type ActionOptsBuilder interface {
 
 // ResourceTag is in key-value format
 type ResourceTag struct {
-	Key   string `json:"key" required:"true"`
```

Comment:
why key not required parameter now?

Reply:
Not necessarily all api calls necessarily require non-null values, this applies to all tag-related operations and requires a certain level of tolerance. but I still add required as you said.